packages/compass-components/package.json

```diff
@@ -37,11 +37,12 @@
     "@leafygreen-ui/card": "^5.1.4",
     "@leafygreen-ui/checkbox": "^6.0.6",
     "@leafygreen-ui/code": "^9.4.0",
+    "@leafygreen-ui/combobox": "^2.0.1",
```

Comment:
This combobox component is implemented in the new brand style, right? Just wanted to double-check whether we are okay with mixing components with new brand styles with the old leafygreen style that Compass is currently using (like this dropdown select that is right after the combobox)

![image](https://user-images.githubusercontent.com/5036933/178311312-e545fb08-66bc-4380-8d99-bfe1b769420c.png)

Reply:
Switched to the older version! Thx for pointing at this 🔥